matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/content/UploadContentWorker.kt

```diff
@@ -280,7 +280,7 @@ internal class UploadContentWorker(val context: Context, params: WorkerParameter
                 }
 
                 // Delete the temporary voice message file
-                if (params.attachment.type == ContentAttachmentData.Type.AUDIO && params.attachment.mimeType == MimeTypes.Ogg) {
+                if (params.attachment.type == ContentAttachmentData.Type.VOICE_MESSAGE) {
```

Comment:
I assume that the file was actually sent but in the UI it appears as an error? Anyway thanks for the fix!

Reply:
the file is uploaded but the associated timeline event is not sent (because we fail before `handleSuccess -> MultipleEventSendingDispatcherWorker`)

so I guess it's floating in limbo